src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.cpp

```diff
@@ -352,6 +356,14 @@ std::string AppleseedObjPropsMod::get_sss_set(const TimeValue t) const
     return str_value != nullptr ? wide_to_utf8(str_value) : std::string();
 }
 
+int AppleseedObjPropsMod::get_medium_priority(const TimeValue t) const
+{
+   int int_value = 0;
```

Comment:
You assign 0 then immediately overwrite the variable.

You can do it all in a single line:

    return m_pblock->GetInt(paramIdNestedmediumPriority, t, FOREVER);

Reply:
Changed it as you have shown.